controllers/storagecluster/cephcluster.go

```diff
@@ -114,6 +114,13 @@ func (obj *ocsCephCluster) ensureCreated(r *StorageClusterReconciler, sc *ocsv1.
 		sc.Spec.StorageDeviceSets[i].Config.TuneSlowDeviceClass = false
 		sc.Spec.StorageDeviceSets[i].Config.TuneFastDeviceClass = false
 
+		// Use SSD deviceClass and TuneFastDeviceClass for all internal clusters where deviceClass is not provided by the customer
+		if !sc.Spec.ExternalStorage.Enable && sc.Spec.StorageDeviceSets[i].DeviceClass == "" {
```

Comment:
@sp98 Can you pls mention in the API as well that if the value is empty it will be considered as SSD. or maybe we use default values kubebuilder tags?

Reply:
Updated the API description for `DeviceClass`.